src/util/FactorUtil.js

```diff
@@ -145,7 +145,7 @@ function (Okta, TimeUtil) {
     if (provider === 'OKTA' && factorType === 'push') {
       return 'OKTA_VERIFY_PUSH';
     }
-    if (provider === 'CUSTOM' && factorType === 'token:software:hotp') {
+    if (factorType === 'token:software:hotp') {
```

Comment:
I think we could put this in `getFactorNameForFactorType`. Webauthn is there also